js/msp/MSPHelper.js

```diff
@@ -1194,6 +1208,27 @@ var mspHelper = (function (gui) {
                 buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_throttle_low_delay));
                 buffer.push(highByte(FAILSAFE_CONFIG.failsafe_throttle_low_delay));
                 buffer.push(FAILSAFE_CONFIG.failsafe_procedure);
+                buffer.push(FAILSAFE_CONFIG.failsafe_recovery_delay);
+
+                if (semver.gt(CONFIG.flightControllerVersion, "1.7.3")) {
+                    buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_fw_roll_angle));
+                    buffer.push(highByte(FAILSAFE_CONFIG.failsafe_fw_roll_angle));
+
+                    buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_fw_pitch_angle));
+                    buffer.push(highByte(FAILSAFE_CONFIG.failsafe_fw_pitch_angle));
+
+                    buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_fw_yaw_rate));
+                    buffer.push(highByte(FAILSAFE_CONFIG.failsafe_fw_yaw_rate));
+
+                    buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_stick_motion_threshold));
+                    buffer.push(highByte(FAILSAFE_CONFIG.failsafe_stick_motion_threshold));
+
+                    buffer.push(lowByte(FAILSAFE_CONFIG.failsafe_min_distance));
```

Comment:
You should protect only `failsafe_min_distance` and `failsafe_min_distance_procedure` with a check for 1.7.4+